editor/project.clj

```diff
@@ -78,7 +78,7 @@
 
                      [org.luaj/luaj-jse "3.0.1"]
 
-                     [cljfx "1.7.19"]
+                     [cljfx "1.7.20"]
```

Comment:
The new cljfx version adds [minor performance improvement](https://github.com/cljfx/cljfx/commit/acbe9972e99abdce2d88d4595aa90f4d7c865629#diff-2d1a8a5d95ca40ee6e203dbca4971239ea263507304181cd86fd28a0bf266deaL44-R46): when performing very first render, if render request happened on UI thread, the render will happen immediately instead of "later".